doc/dbext.txt

```diff
@@ -117,6 +118,72 @@ David Fishburn
 ==============================================================================
 2. What's New						*dbext-new*
 
+Version 26.00 (Nov 17, 2017)
```

Comment:
Can you add all the new features you put in, new variables to customize, and new Vim functions to be called.